zenoh-flow/src/runtime/runners/operator.rs

```diff
@@ -23,73 +23,88 @@ use libloading::Library;
 use std::collections::HashMap;
 use uhlc::HLC;
 
-pub type ZFOperatorRegisterFn = fn() -> ZFResult<Box<dyn ZFOperatorTrait + Send>>;
+pub type ZFOperatorRegisterFn = fn() -> ZFResult<Arc<dyn ZFOperatorTrait>>;
 
 pub struct ZFOperatorDeclaration {
     pub rustc_version: &'static str,
     pub core_version: &'static str,
     pub register: ZFOperatorRegisterFn,
 }
 
+pub type ZFOperatorIO = (
+    Vec<ZFLinkReceiver<ZFMessage>>,
+    HashMap<String, Vec<ZFLinkSender<ZFMessage>>>,
+);
+
+#[derive(Clone)]
```

Comment:
Add comment to not change the order of the struct's fields, same in SinkRunner and SourceRunner